Packs/Symantec_Deepsight/Integrations/Symantec_Deepsight/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
--
+Updated the server url to the new Symantec Deepsight service, no changes to default behaviour are expected.
```

Comment:
Updated the server url to the new Symantec Deepsight service. There are no expected changes to default behaviour.